mlrun/__main__.py

```diff
@@ -1104,19 +1102,17 @@ def project(
                     "token": proj.get_param("GIT_TOKEN"),
                 },
             )
-        if notification:
-            notifications = line2keylist(notification, keyname="type", valname="params")
+        if notifications:
             for notification in notifications:
-                if notification["type"] == "file":
-                    with open(notification["params"]) as fp:
-                        lines = fp.read().splitlines()
-                        notification_from_file = list2dict(lines)
+                if "=" in notification:
```

Comment:
what is it "=" in notification?
there are many logic here and I can't follow. do you mind to comment or explain what is that doing here?

Reply:
Added a comment, If something is not clear, please let me know.